samples/Azure.Management.Storage/Generated/BlobContainerCollection.cs

```diff
@@ -56,8 +56,16 @@ internal static void ValidateResourceId(ResourceIdentifier id)
 
         /// <summary>
         /// Creates a new container under the specified account as described by request body. The container resource includes metadata and properties for that container. It does not include a list of the blobs contained by the container. 
-        /// Request Path: /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/blobServices/default/containers/{containerName}
-        /// Operation Id: BlobContainers_Create
+        /// <list type="bullet">
+        /// <item>
+        /// <term>Request Path</term>
+        /// <description>/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/blobServices/default/containers/{containerName}</description>
```

Comment:
Should we add an extra space in the beginning and in the end to follow existing doc format?

Reply:
> Should we add an extra space in the beginning and in the end to follow existing doc format?

@fengzhou-msft It's not a new line. It's the formatting problem in github diff view. The line is too long so that the wrapping is not correct. Below is what the original source codes look like:
![image](https://user-images.githubusercontent.com/4516/211513545-d3fe5486-8fa7-4b71-bb2f-b203f28bdb08.png)